workflow/local.smk

```diff
@@ -37,3 +43,17 @@ rule plot:
         gammapy_env
     shell:
         "MATPLOTLIBRC={input.rc} python {input.script} -i {input.data} -o {output}"
+
+
+# Extra rule because there is one script generating many plots with differing names
+rule plot_theta:
+    output:
+        "build/plots/{analysis}/theta2_{runid}.pdf",
+    input:
+        data="build/dl4/{analysis}/theta2_{runid}.fits.gz",
+        script="scripts/plot_theta2.py",
```

Comment:
this script plots all of them, doesn't it? so either the wildcards are wrong or the comment. I'm confused

Reply:
lets discuss offline